hubspot-client-bundles/hbase-backup-restore-bundle/pom.xml

```diff
@@ -65,6 +65,12 @@
           <execution>
             <id>create-bundle-with-relocations</id>
             <configuration>
+              <transformers>
+                <!-- carry over hbase-site.xml from hbase-mapreduce-bundle -->
+                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
```

Comment:
This may not be necessary at all actually. We list hbase-mapreduce-bundle as a dependency, so if someone tried pulling this bundle into a deployable they'd transitively get hbase-mapreduce-bundle which would include the hbase-site.xml in it. Does that sound right?

Reply:
It sounds right, but after doing it and inspecting the resulting JARs, it doesn't happen

Reply:
(deleted previous comment because I read your note too fast) I think you're right, that could work okay.